modules/apps/commerce/commerce-service/service.xml

```diff
@@ -402,6 +402,9 @@
 		<finder name="CProductId" return-type="Collection">
 			<finder-column name="CProductId" />
 		</finder>
+		<finder name="CustomerCommerceOrderItemId" return-type="Collection">
```

Comment:
wrong commit name, you are adding a finder not a column, that one was already added before